example/index.js

```diff
@@ -70,6 +70,22 @@ db.installUsersBehavior().then(function () {
     }
   })
 
+  server.route({
+    method: 'GET',
+    path: '/default/css/default.css',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.css'))
+    }
+  })
+
+  server.route({
+    method: 'GET',
+    path: '/default/js/default.js',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.js'))
+    }
+  })
```

Comment:
Yes we use inert in the [main `hoodie` repository](https://github.com/hoodiehq/hoodie/blob/master/server/plugins/public.js) to serve static assets from the Hoodie app’s `public/` folder